style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif, 'poppins';
+  font-size: 100%;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.flex-div {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 7%;
+  box-shadow: 0 0 0.62rem rgba(0, 0, 0, 0.2);
+}
+
+.nav-left .logo {
+  font-size: 1.12rem;
+  color: #6070ff;
+  margin-left: 1.5rem;
+  margin-top: 0.87rem;
+  margin-bottom: 0.87rem;
+}
+
+.nav-right img {
+  padding-top: 1.18rem;
+  padding-right: 2.25rem;
+}
+
+.main-container {
+  width: 100%;
+  height: 100%;
+  background-image: url(src/img/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.greeting-text h2 {
+  font-size: 2.5rem;
+  width: 80%;
+  height: 20%;
+  font-weight: 43.75rem;
+  line-height: 3.25rem;
+  color: #172b4d;
+}
+
+.greeting-text p {
+  font-size: 1rem;
+  width: 90%;
+  height: 20%;
+  line-height: 1.5rem;
+  font-weight: 25rem;
+  color: #344563;
+  padding-top: 2%;
+}
+
+.main-container .greeting-text {
+  padding-top: 30%;
+  padding-left: 7%;
+  padding-right: 7%;
+}
```

Comment:
- [ ] Kindly consider laying out your elements in hero section using display: flex as per requirements as it helps in the responsive design.
  - In order to lay out the elements on the page you should use flexbox in all 2 sections.